metadata/index.html

```diff
@@ -2177,19 +1936,11 @@
       </p>
     </section>
     <section class="appendix" id="changes">
-      <h2>Changes since the working draft of 08 January 2015</h2>
-      <p>The document has undergone substantial changes since the last working draft. Below are some of the changes made:</p>
+      <h2>Changes since the working draft of 16 April 2015</h2>
```

Comment:
I think it is worth keeping the previous set of changes and add these as separate entries. If somebody wants to follow up history, it is more convenient to find everything at one place (a comment we did get when we published RDFa Edited Rec...)